### ISSUE TITLE
Support InternImage Model

### ISSUE THREAD

**Issue by M-Colley** [open-mmlab/mmsegmentation#3126]
It would be great to also support current SOTA models on semantic segmentation, especially https://github.com/opengvlab/internimage would be great for my specific use case.

Thanks for this great library!

Kind regards

**Comment by Pascal-Jansen** [comment]
I agree with @M-Colley that supporting newer models on semantic segmentation would further improve this already awesome repository.

Thanks in advance for considering this!
